bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -5,5 +5,6 @@
 
 	<name>Pentair Binding</name>
 	<description>This is the binding for Pentair pool systems.</description>
+	<author>Jeff James</author>
```

Comment:
Please remove this. This tag has been deprecated and removed from other bindings.